.github/workflows/image-deps-updater.yaml

```diff
@@ -13,8 +13,10 @@ jobs:
   update-k0s-images:
     runs-on: ubuntu-20.04
     steps:
-    - name: Checkout
```

Comment:
im now surprised this works without checking out

Reply:
Oops, it probably doesn't. It didn't run in CI yet.